recipe/meta.yaml

```diff
@@ -19,9 +19,11 @@ source:
     # - https://bugs.python.org/issue28708
     # - https://github.com/zeromq/libzmq/issues/1165
     - fd_setsize.patch                # [win]
+    - mingw-w64--DMS_WINXX.patch
+    - win-msc_ver-1900.patch
 
 build:
-  number: 1
+  number: 2
```

Comment:
Github is playing with us. This was `2` before and there should be no change here. Build number `2` was set in this PR https://github.com/conda-forge/python-feedstock/pull/139/files.

cc @ocefpaf